source/includes/usage-examples/code-snippets/bulk.go

```diff
@@ -45,7 +43,6 @@ func main() {
 		}
 	}()
 
-	// begin bulk
 	coll := client.Database("sample_restaurants").Collection("restaurants")
 
 	// Creates write models that specify replace and update operations
```

Comment:
S: I know this is a pre-existing ex, but it's a bit confusing to add the new Cafe Zucchini doc and then also update it in one bulk operation without showing what the new documents look like. 
Perhaps change the Cafe Zucchini name in the the replacement doc to something else to avoid confusion? Another possibility is to show what the documents look like now as a result of the bulk operation.

Reply:
Thanks for updating! I think it would make more sense to update a completely new document in the second operation, since the cuisine can also be updated in the first replace operation.